db/db.js

```diff
@@ -21,13 +21,16 @@ const { Schema } = mongoose;
 const productSchema = new Schema({
   id: Number,
   product_name: String,
-  prime_pic: String,
   price: String,
   seller: String,
   rating: Number,
   shipping: Boolean,
   condition: Boolean,
   availability: Number,
+  review_count: Number,
+  sale_count: Number,
+  sellerRating_count: Number,
+  seller_feedback: String,
```

Comment:
added this missing data to database

Reply:
Looks Good!
Are you planning on working with miles on getting the same seller rating and feedback?
Beyond that looks good to merge!

Reply:
I think the only data I will be getting from miles is the number of reviews, which is why they are all 0 for now. Everything else will be randomized!